simple/src/codingPractice/SortArray.java

```diff
@@ -0,0 +1,25 @@
+package codingPractice;
+
+public class SortArray {
+    public static void main(String[] args) {
```

Comment:
There should be space in between class and method.

Reply:
done